styles/style.css

```diff
@@ -88,33 +103,44 @@ body {
 	max-width: 100px;
 	background: #ce5f31;
 }
-
+.container {
+	flex: 1 1 30%;
+}
+.box {
+  height: 150px;
+  width: 150px;
```

Comment:
⚠️ You use the `box` class on the same element as a `col-md-3` class... 

https://github.com/CodeYourFuture/bikes-for-refugees/pull/32/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR67

which one will win?